### PR TITLE
added unflag-tooltip to claify button

### PR DESCRIPTION
**Pull request by stephaniequintana**
Fixes #10839 (the PR fix for the original issue) and #10820 (the original issue) 

added `data-toggle="tooltip" data-placement="top" title="Remove all flags"`
to the existing "unflag" button:
```
<a class="btn btn-xs border-curve font-weight-bold btn-warning unflag"
  data-remote="true"href="/moderate/remove_flag_node/<%= flag.id %>">Unflag</a>
```
The changes will be applied to the following:

Screen Shot 2022-03-30 at 2 49 41 PM

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [X] PR is descriptively titled 📑 and links the original issue above 🔗
* [X] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [X] code is in uniquely-named feature branch and has no merge conflicts 📁
* [X] screenshots/GIFs are attached 📎 in case of UI updation
* [X] ask `@publiclab/reviewers` for help, in a comment below

This PR was created in reference for the original PR, #10839, as the branch (and corresponding repo) has been deleted and is unrecoverable.

### CODE REVIEW

File: app/views/spam2/_flags.html.erb [hunk 1]
```diff
@@ -110,7 +110,7 @@ $(document).ready(function () {
              <td style="height:35px !important;">
                <a class="btn btn-xs border-curve font-weight-bold btn<% if flag.status != 1 %>-success<% else %>-secondary disabled<% end %> publish" data-remote="true" href="/moderate/publish/<%= flag.id %>" ><i class="fa fa-check-circle fa-white"></i> Publish post</a>
                <a class="btn btn-xs border-curve font-weight-bold btn<% if flag.status != 0 %>-danger<% else %>-secondary disabled<% end %> spam" data-remote="true" href="/moderate/spam/<%= flag.id %>"><i class="fa fa-ban fa-white"></i> Spam post</a> 
-               <a class="btn btn-xs border-curve font-weight-bold btn-warning unflag" data-remote="true"href="/moderate/remove_flag_node/<%= flag.id %>">Unflag</a>
+               <a data-toggle="tooltip" data-placement="top" title="Remove all flags" class="btn btn-xs border-curve font-weight-bold btn-warning unflag" data-remote="true"href="/moderate/remove_flag_node/<%= flag.id %>">Unflag</a>
```
Comment (TildaDares) [inline app/views/spam2/_flags.html.erb:113]:
```suggestion
               <a id="unflag-node" data-toggle="tooltip" data-placement="top" title="Remove all flags" class="btn btn-xs border-curve font-weight-bold btn-warning unflag" data-remote="true"href="/moderate/remove_flag_node/<%= flag.id %>">Unflag</a>
```

Comment (TildaDares) [reply app/views/spam2/_flags.html.erb:113]: And then on line 46, add the following code:
`$('#unflag-node').tooltip({ boundary: 'window' })`

Comment (TildaDares) [reply app/views/spam2/_flags.html.erb:113]: Let's hope that works!

Comment (TildaDares) [reply app/views/spam2/_flags.html.erb]: It worked🎉🎉🎉🎉

Comment (stephaniequintana) [reply app/views/spam2/_flags.html.erb]: This is fantastic, finally! Thank you, Tilda 😄

Comment (TildaDares) [reply app/views/spam2/_flags.html.erb]: I noticed that the tooltip kept flickering when I hovered over it so I searched for that and this was the SO answer that helped https://stackoverflow.com/a/56762953/11953728

**Comment by stephaniequintana** [comment]
Resolved #10839 (the PR fix for the original issue) and #10820 (the original issue)

**Comment by TildaDares** [comment]
Hi @stephaniequintana, why did you close this issue?

**Comment by stephaniequintana** [comment]
@TildaDares, I did - my apologies. When you asked that I close #10839 because this one resolved the issue, I see that I had mistakenly closed this one as well. Shall I reopen it now?

**Comment by TildaDares** [comment]
Sure, you can reopen it.

**Comment by stephaniequintana** [comment]
Reopening due to premature close.

**Review by TildaDares — APPROVED** [review]

**Comment by TildaDares** [comment]
Congratulations @stephaniequintana 🎉

**Comment by stephaniequintana** [comment]
Thank you, @TildaDares - that was a journey! So glad to finally be a "contributor"!! 🥳 🎆

**Comment by TildaDares** [comment]
@stephaniequintana Coding is not the only form of contribution. Your support for other applicants and creating FTOs are all valid contributions ❤️

**Pull request merged**